instance/migrations/0025_auto_20150920_0907.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import instance.models.utils
+import django.db.models.fields
+import django_extensions.db.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0024_auto_20150911_2304'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='GeneralLogEntry',
+            fields=[
+                ('id', models.AutoField(serialize=False, auto_created=True, verbose_name='ID', primary_key=True)),
+                ('created', django_extensions.db.fields.CreationDateTimeField(default=django.db.models.fields.NOT_PROVIDED, verbose_name='created', auto_now_add=True)),
+                ('modified', django_extensions.db.fields.ModificationDateTimeField(auto_now=True, default=django.db.models.fields.NOT_PROVIDED, verbose_name='modified')),
```

Comment:
@mtyaka This comes from the `TimeStampedModel` mixin from [Django extensions](http://django-extensions.readthedocs.org/en/latest/model_extensions.html), it's present on all models. Log entries aren't expected to change, no, but the field doesn't seem harmful either - creating a new mixin without the modified field to replace https://github.com/django-extensions/django-extensions/blob/199e659fd8339d09e809d257f485eeda1b7f9e93/django_extensions/db/models.py#L19 might be more trouble that it's worth.